docs/install/management-address.md

```diff
@@ -6,7 +6,7 @@ Description: The Harvester provides a virtual IP as the management address.
 ---
 
 # Management Address
-Harvester provides a fixed virtual IP (VIP) as the management address. You can find the management address on the console dashboard after the installation.
+Harvester provides a fixed virtual IP (VIP) as the management address, VIP should be different from any node IP.  You can find the management address on the console dashboard after the installation.
```

Comment:
Please unify  `should be` /  `must be` ,  `different from` / `different than` and `node IP` / `NODE IP` ind all the changed files, thanks.